meson.build

```diff
@@ -11,6 +11,9 @@ project(
     ],
 )
 
+ROOT_DIR = meson.source_root()
+BUILD_DIR = meson.build_root()
```

Comment:
Meson certainly doesn't require it, you can just use the functions inline.

Although anyway those functions are deprecated in favor of the more explicit `global_source_root()` / `global_build_root()` functions, and discouraged either way because most people "probably" want `project_source_root()` / `project_build_root()`.

Reply:
Okay, thank you. Done.